bin/cron/delete_old_ai_chat_data

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+require 'cdo/chat_client'
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+# We want to avoid storing student chat messages any longer than
+# is necessary for analysis to prevent storing excess student PII. This deletes
+# any AichatEvent or AichatRequest older than 90 days.
+# 
+# Additionally, we delete any AichatSession older than 30 days. AichatSession is
+# no longer in use and we plan to fully delete the table.
+# See https://codedotorg.atlassian.net/browse/LABS-976 for details.
```

Comment:
Oh oops I think I misspoke over Slack - I think 90 days (same as the other two) should also work for AichatSessions for now? That way we still have a bit of time to review any last minute data from our July pilot if necessary